.env.development

```diff
@@ -1,3 +1,3 @@
 VUE_APP_MODE = development
-VUE_APP_BACKEND_PREFIX = http://localhost:5000
+VUE_APP_BACKEND_PREFIX = http://localhost:8080
```

Comment:
shouldn't this still be port 5000?

Reply:
Oh, that's a remnant of the CORS issue. That's how we overcame it. I'm not sure whether this is an actual issue or a misconfiguration. We need to test this on another environment.